### PR TITLE
feat(general) : MP-232 Create Form Style

### PR DESCRIPTION
**Pull request by RTAgung**
## Update Items
- [x] [[MP-232](https://args06.atlassian.net/browse/MP-232)] Create Form Style

## Summary Description
- Create form style in theme.xml
- Create new state color

## *Type of Change
- [ ] docs: Documentation only changes, README.md, Wiki
- [x] feat: A new feature
- [ ] refactor: A code change that neither fixes a bug nor adds a feature
- [ ] test: Anything related to unit / integration testing
- [ ] revert: Revert a commit

## Type of Scope
- [ ] album: Changes that affect user packages
- [ ] music: Changes that affect music packages
- [ ] account: Changes that affect account packages
- [ ] favorite: Changes that affect favorite packages
- [ ] playlist: Changes that affect playlist packages
- [ ] playlist-detail: Changes that affect playlist-detail packages
- [ ] music-artist: Changes that affect music-artist packages
- [x] general: all stuff except above

### CODE REVIEW

File: app/src/main/res/color/text_color_hint.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (args06) [inline app/src/main/res/color/text_color_hint.xml:2]: Kenapa ga pake style untuk style text hint?

**Review by args06 — CHANGES_REQUESTED** [review]

File: app/src/main/res/color/text_color_hint.xml (hunk 1 above)
Comment (RTAgung) [reply app/src/main/res/color/text_color_hint.xml:2]: what do you mean?

Comment (args06) [reply app/src/main/res/color/text_color_hint.xml:2]: Ini itu buat style dri textnya kan? Kenapa ga di style nya Theme

Comment (RTAgung) [reply app/src/main/res/color/text_color_hint.xml:2]: gaa itu utk ngatur warna nya doang biar dinamis ada statenya

Comment (args06) [reply app/src/main/res/color/text_color_hint.xml:2]: oh ok

**Review by args06 — APPROVED** [review]

**Pull request merged**